virttest/qemu_capabilities.py

```diff
@@ -19,6 +19,7 @@ class Flags(object):
     """ Enumerate the flags of VM capabilities. """
 
     BLOCKDEV = _auto_value()
+    CPUDIE = _auto_value()
```

Comment:
It should be a parameter like `socket-id`, `core-id` that defines the cpu topology rather than the smp architecture.
So I think `CPU_DIES` might be more suitable.

Reply:
updated